src/styles/main.scss

```diff
@@ -30,6 +30,11 @@ a {
     text-decoration: none;
 }
 
+.text-content {
```

Comment:
1200px is was (at least at one point) supposed to be the width of anything on the site that wasn't a hero-type banner. Because having text the full width of a wide screen makes it cumbersome to read.